v3ioutils/container.go

```diff
@@ -145,10 +145,12 @@ func DeleteTable(logger logger.Logger, container v3io.Container, path, filter st
 			}
 		case err := <-deleteTerminationChan:
 			if err != nil {
-				for i := 0; i < 2*workers; i++ {
-					onErrorTerminationChannel <- struct{}{}
+				if errorWithStatusCode, ok := err.(v3ioerrors.ErrorWithStatusCode); !ok || !ignoreMissing || (errorWithStatusCode.StatusCode() != http.StatusNotFound && errorWithStatusCode.StatusCode() != http.StatusConflict) {
```

Comment:
1. use `utils.IsNotExistsOrConflictError(err)` instead of explicitly checking error codes.
2. In this place you will ignore those errors but you stop deleting remaining object in the "bad" partition. i think we better ignore the errors further down at `deleteEntirePartition` and `utils.DeleteTable`